types/node/index.d.ts

```diff
@@ -756,20 +756,23 @@ declare module "http" {
         write(str: string, encoding?: string, cb?: Function): boolean;
         write(str: string, encoding?: string, fd?: string): boolean;
 
-        writeContinue(): void;
-        writeHead(statusCode: number, reasonPhrase?: string, headers?: any): void;
-        writeHead(statusCode: number, headers?: any): void;
-        statusCode: number;
-        statusMessage: string;
+        addTrailers(headers: any): void;
+        finished: boolean;
+        getHeader(name: string): string;
+        getHeaderNames(): string[]
+        getHeaders(): any
```

Comment:
The interpretation of just an interface can be different depending on the language you are used to. In TypeScript it inherits from Object. I don't think it is documented that specific, but it is easy to test.